src/lib.rs

```diff
@@ -87,8 +89,8 @@ impl<'a> Job<'a> {
         Job {
             schedule,
             run: Box::new(run),
-
             last_tick: None,
+            job_id: Uuid::new_v4().to_string(),
```

Comment:
Sorry about sending the review a day later. Forgot to actually submit the review itself :)

Reply:
I thought a String might be more familiar to work with than Uuid, even though it means it would take up twice as much memory space.

I didn't realize that Uuid implemented copy. I'll change it once I get back to my desk.

Reply:
Updated! 😀